spec/maventest_spec.vim

```diff
@@ -18,11 +19,34 @@ describe "Maven"
     Expect g:test#last_command == 'mvn test -Dtest=MathTest'
   end
 
+  it "runs file tests with user provided options"
+    view MathTest.java
+    let g:test#java#maventest#options = '-f pom.xml'
+    TestFile
```

Comment:
Instead of assigning the `g:test#java#maventest#options` variable, you can pass these options directly to the command:

```vim
view MathTest.java
TestFile -f pom.xml
```

All test.vim commands accept command line options that they just forward to the test runner (they get merged into to `a:args` in `test#{language}#{runner}#build_args`), in the same way that that options from the `g:test#{language}#{runner}#options` variable get merged. In other words, these two ways of passing arguments are equivalent.